src/Quasar/Advanced/Types.purs

```diff
@@ -464,6 +464,37 @@ instance encodeJsonProvider ∷ EncodeJson Provider where
     ~> "client_id" := Newtype.unwrap obj.clientId
     ~> "openid_configuration" := OpenIDConfiguration obj.openIDConfiguration
     ~> jsonEmptyObject
+    
+data LicenseStatus = LicenseValid
+
+decodeLicenseStatus ∷ Json → Either String LicenseStatus
+decodeLicenseStatus =
+  (if _ then Right LicenseValid else Left $ "status != " <> show licenseValid)
+    <<< eq licenseValid
+    <=< decodeJson
+  where
+  licenseValid = "LICENSE_VALID"
+
+type LicenseInfo'
+  = { expirationDate ∷ String
```

Comment:
Not sure how to parse a date without the `LOCALE` effect. Easy enough to parse it elsewhere.